src/components/DetailedPodcast/DetailedPodcast.spec.js

```diff
@@ -0,0 +1,10 @@
+import {render, screen} from '@testing-library/react'
+import DetailedPodcast from './DetailedPodcast'
+  
+describe('DetailedPodcast', () => {
+    it('renders', () => {
+        render(<DetailedPodcast />)
+        expect(screen.getByText('DetailedPodcast')).toBeInTheDocument()
+    })
+})
+      
```

Comment:
This test might not be necessary since you only check rendering of the component, not any props passed. Speaking of which - you pass podcast to the component you test here. Maybe add a fake podcast object and check if it renders properly with the specific values.

Reply:
Yes will do so. I will do a day of updating all test in the near future :)